### DISCUSSION TITLE
Is there optimized action lists for Arms Warrior?The default list given by Simc doesn't seem to be optimal.

### DISCUSSION THREAD

**Discussion by GonzoXing** [simulationcraft/simc#6178] [Q&A]
I need a better action list for Arms Worrior,Can someone help me?

**Comment by scamille** [comment]
Actions lists are part of the input you provide to SimulationCraft. You can always change them yourself and try to improve them, to either get more DPS, better reflect how the class is actually played or to improve different fight style components like multi target or movement.

The default action lists in SimulationCraft strive to give the "best" results for each class, but they are always limited by whoever is working on them. Some classes and specs just have more active developers and other people interested in them.

If you come up with improvements, you can create a issue with your suggestions, if possible backed by some simulation results (Raidbot links are a great way to do that).

That said, you might also find more help and information directly in a Discord for your class. There is usually a community to be found somewhere for each WoW class.
